resources/prosody-plugins/token/util.lib.lua

```diff
@@ -33,6 +33,7 @@ function Util.new(module)
     self.appId = module:get_option_string("app_id");
     self.appSecret = module:get_option_string("app_secret");
     self.asapKeyServer = module:get_option_string("asap_key_server");
+    self.signatureAlgorithm = module:get_option_string("signature_algorithm");
```

Comment:
Now there is an extra configuration that we depend on, without default value and all the users that update will be broken till they update their config? 
I also do not see this option added to the default prosody template that is used for new installs.
Why is this needed to be configurable? Isn't it that when you use a shared secret (module:get_option_string("app_secret");) it is HS256 and when using "asap_key_server" it is RS256?

Reply:
> Isn't it that when you use a shared secret (module:get_option_string("app_secret");) it is HS256 and when using "asap_key_server" it is RS256?
luajwtjitsi supports larger key sizes (e.g. HS512/RS512) which I would definitely recommend you to support. But it is probably good to have it default to HS256/RS256 when `app_secret`/`asap_key_server` is set, and then allow to overwrite when `signature_algorithm` is set. I will update with that change.

Reply:
I have added defaults for the `signature_algorithm` option based on the user of `app_secret`/`asap_key_server`. That should preserve the current functionality.